### PR TITLE
Locked npm version in the Dockerfile - Closes #491

### LINKED ISSUE #491: Dependences review
According to Snyk report, there are few dependencies that require updating.
Following actions need to be taken:

Medium severity:
- [x] Upgrade to npm@5.7.1

Low severity:
- [ ] Upgrade to lodash@4.17.5
- [ ] Upgrade to z-schema@3.19.0.
- [ ] Upgrade to validator@9.4.1

### PR DESCRIPTION
**Pull request by MichalTuleja**
Current update forces Docker build container to use npm version 5.7.1
### Review checklist
- The PR solves #491
- All new code follows best practices

### CODE REVIEW

**Comment by fchavant** [comment]
@MichalTuleja `npm` comes from the official `node` image (first line of the `Dockerfile`) -- have you tried pulling the newest version?

**Comment by MichalTuleja** [comment]
I'd rather stay with the version of Node that we support.
The upgrade of npm is justified by the latest Snyk report.

File: Dockerfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ COPY . /home/lisk/lisk-explorer/
 RUN useradd lisk && \
     chown lisk:lisk -R /home/lisk
 USER lisk
+RUN npm install -g npm@5.7.1
```
Comment (fchavant) [inline Dockerfile]: Let's move this up to line 8 instead of having it in its own layer.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -4,6 +4,7 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get --assume-yes install --no-install-recommends \
         build-essential \
 	redis-server && \
+    npm install -g npm@5.7.1 \
```
Comment (fchavant) [inline Dockerfile]: `&&` missing

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**